contributing/ORIGIN_TRIALS.md

```diff
@@ -0,0 +1,22 @@
+# Origin trials
+Origin trials enable developers to use an experimental feature in production and provide essential feedback.
+
+The AMP Origin Trial process is inspired by the [Chromium Origin Trial process](https://github.com/GoogleChrome/OriginTrials). 
+
+Traditionally, a feature in experimental mode can be used in development, but cannot be pushed to production. With Origin trials, interested developers can opt-in to a test an experimental feature in production, with the following expectations:
+- The test is for a limited time.
+- The feature will likely undergo some changes after an Origin Trials.
+- Origin trials present an opportunity to implement and benefit from a new feature before it’s fully live. The feature will live on the developer's site, rather than guarded by an experiment, and feedback can directly influence the direction of the feature.
+
+## When to consider an Origin Trial
+
+## Origin Trial - the details
+- No Origin Trial can last more than 3 months.
+- People generating Origin Trial tokens should be making sure that no more than 0.1% of AMP page loads (the deprecation threshold) starts depending on an Origin Trial
+- Extending an Origin Trial requires approval from the TSC. They can delegate this to the Working Group that is responsible for the Origin Trial.
```

Comment:
Should we consider delegating this to the applicable Working Groups by default?

Reply:
I thought this was supposed to be a high bar, so people should not be able to approve themselves. I think we may actually want to remove the "can delegate to the working group", but instead delegate to the Approvers Group.

Reply:
Are you suggesting delegating extension approvals to the Approvers Group in general? Or are you suggesting merely that the TSC can delegate to the Approvers Group (perhaps case by case)?

Reply:
What I wanted to clarify:

Origin trials are meant to be very time-bound and generally should NOT be extended in time. So, the group that wants the trial to happen (the WG responsible for the trial) isn't a good decider to . extend it. 

I'm not sure what the text wanted to say, but to me it sounds like it is suggested to delegate decision making to the WG that requested the OT.

The Approvers group is a better delegate, since they aren't directly involved in the trial.

Reply:
@nainar Let's change this to:

"Extending an Origin Trial requires approval from the Approvers Working Group (@ampproject/wg-approvers)."

Reply:
Done inline.